app/controllers/PayPalOneOff.scala

```diff
@@ -104,7 +114,7 @@ class PayPalOneOff(
 
     emailForUser(request.user)
       .flatMap(paymentAPIService.executePaypalPayment(paymentJSON, acquisitionData, queryStrings, _, isTestUser))
-      .fold(resultFromPaymentAPIError, resultFromPaypalSuccess)
+      .fold(resultFromPaymentAPIError, success => resultFromPaypalSuccess(success, country))
```

Comment:
I think we may need to update the error page too eventually but I don't think we've even got a design for that (and it's non-essential for launching the test). I've added a card for it: https://trello.com/c/je1lyCWY/99-npf-follow-up-paypal-error-page-design-for-new-flow